src/App.jsx

```diff
@@ -14,57 +15,53 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [good, setGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const handleSelect = item => {
+    setGood(prevGood => (prevGood === item ? '' : item));
```

Comment:
The logic for toggling the selected good is correct, but it might be clearer to explicitly handle the deselection case by checking if `item` is an empty string.